.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+# See reference docs at
+# https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Clone the repo
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: vars
+        run: echo "---${{github.head_ref}}---"
```

Comment:
One thing I'm not sure of is whether `github.head_ref` is the same as `TRAVIS_BRANCH` when a PR is sent from another repo, but I guess we'll find out soon.